confluent_server/confluent/auth.py

```diff
@@ -115,6 +116,31 @@ class PromptsNeeded(Exception):
     def __init__(self, prompts):
         self.prompts = prompts
 
+ #add function to change _allowedbyrole and _deniedbyrole vars.
+ def add_roles(dictionary):
+    #function to parse the roles and the files. If there are modifications to be done to the roles, items will be added to dictionaries.
+    #If there are no moodifications done to one of the roles, it continues to the next
+    #Opening YAML file and reading the custom roles
+    with open("/etc/confluent/authorization.yaml","r") as stream:
+        loaded_file = yaml.safe_load(stream)
+        try:
+            dictionary.update(loaded_file)
+        except FileNotFoundError:
+            return "File does not exist"
+        return
+                
+    
+def check_for_yaml():
+    #checking if the file exists
+    if exists("/etc/confluent/authorization.yaml"):
+        add_roles(_allowedbyrole)
+        add_roles(_deniedbyrole)
```

Comment:
This looks like it would load the exact same structure into both _allowedbyrole and _deniedbyrole, meaning it can never be used to create a role that is allowed to do anything...